diff --git a/node_modules/discord.js/src/client/ClientDataManager.js b/node_modules/discord.js/src/client/ClientDataManager.js
index 4f0f2d7..7d33451 100644
--- a/node_modules/discord.js/src/client/ClientDataManager.js
+++ b/node_modules/discord.js/src/client/ClientDataManager.js
@@ -78,7 +78,7 @@ class ClientDataManager {
             break;
         }
 
-        guild.channels.set(channel.id, channel);
+        channel?.id && guild.channels.set(channel.id, channel);
       }
     }
 
